src/gui_common/tooltip/microbe_editor/SelectionMenuToolTip.cs

```diff
@@ -268,6 +268,86 @@ public void WriteOrganelleProcessList(List<ProcessSpeedInformation> processes)
         }
     }
 
+    public void WriteMembraneModifierList(MembraneType referenceMembrane, MembraneType membraneType)
+    {
+        string[] modifierNames =
+        {
+            "Mobility", "Osmoregulation Cost", "Resource Absorption Speed", "Health",
+            "Physical Resistance", "Toxin Resistance",
+        };
```

Comment:
Also I'm wondering if the modifiers of a membrane type should be put inside an array in the membranes.json (and each modifier entry contains its display name and value, might need a MembraneModifier object to hold those?) so we could loop through from a list of MembraneModifiers in a membrane type instead. Not sure if that's necessary though

Reply:
It seems like this list is here, but then there is a special case used in the foreach loop?
To me this list looks like it shouldn't even exist.